mozilla_django_oidc/contrib/drf.py

```diff
@@ -79,7 +79,7 @@ def authenticate(self, request):
             # we can get from the www-authentication header) in the response.
             if resp.status_code == 401 and 'www-authenticate' in resp.headers:
                 data = parse_www_authenticate_header(resp.headers['www-authenticate'])
-                raise exceptions.AuthenticationFailed(data['error_description'])
+                raise exceptions.AuthenticationFailed(data.get('error_description'))
```

Comment:
Could that be:
```suggestion
                raise exceptions.AuthenticationFailed(data.get('error_description') or 'no error_description in www-authenticate')
```
just so that you at least can't end up with a `raise exceptions.AuthenticationFailed(None)`

Reply:
or something like Peter's suggestion with `get`

```python
data.get('error_discription', 'no error description in www-authenticate')
```